modules/gcp/region.go

```diff
@@ -186,3 +213,17 @@ func GetAllGcpZonesE(t *testing.T) ([]string, error) {
 
 	return zones, nil
 }
+
+// Given a GCP Zone URL formatted like https://www.googleapis.com/compute/v1/projects/project-123456/zones/asia-east1-b,
+// return "asia-east1-b".
+func ZoneUrlToZone(zoneUrl string) string {
+	tokens := strings.Split(zoneUrl, "/")
+	return tokens[len(tokens)-1]
```

Comment:
To avoid really confusing errors—e.g., I could call this method with "foo" or "foo/bar" and still get back data—should this be a regex to do a bit more sanity checking?

Reply:
Good call. I'm unfortunately out of time, so I've left a ToDo for now.